backend/src/main/java/com/fit/nufit/meal/dto/response/MealDetailsResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.fit.nufit.meal.dto.response;
+
+import com.fit.nufit.food.dto.FoodSimpleResponse;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+public class MealDetailsResponse {
+
+    private Long id;
+    private int calorieTotal;
```

Comment:
변수명을 `totalCalorie`로 바꾸는 것이 어떨까요? 😁
```suggestion
    private int totalCalorie;
```

Reply:
제가 공부하다 보니까 한정자는 뒤쪽에 붙여서 통일시키는 것이 컨벤션이라고 합니다! 저희도 한정자는 뒤로 빼는 것으로 리팩토링 진행하면 좋을 것 같아요 😁

https://velog.io/@chosj1526/%EC%A2%8B%EC%9D%80-%EC%BD%94%EB%93%9C%EB%A5%BC-%EC%9C%84%ED%95%9C-JAVA-%EB%B3%80%EC%88%98-%EB%84%A4%EC%9D%B4%EB%B0%8D